pkg/kubectl/cmd/logs.go

```diff
@@ -224,6 +225,11 @@ func (o LogsOptions) Validate() error {
 		return errs.ToAggregate()
 	}
 
+	var err error
+	o.GetPodTimeout, err = cmdutil.GetPodRunningTimoutFlag(cmd)
```

Comment:
This fits better in `Complete` instead of here in `Validate`. In which case you'll not need to pass `cmd` to the `Validate` method.